test/collection/lru_map_test.dart

```diff
@@ -36,6 +36,7 @@ void main() {
 
       expect(lruMap.keys.toList(), ['C', 'B', 'A']);
 
+      // ignore: unnecessary_statements
       lruMap['B'];
```

Comment:
You can avoid `// ignore` with:

```dart
final _ = lruMap['B'];
```

Reply:
Either that or add a short comment just before the `// ignore` explaining why it's necessary.

Reply:
TIL :) Thanks! I've replaced the `// ignore:` with comments explaining the intent, which I admit wasn't entirely obvious to me on at least one of these. Leaving a mental note to go back and document that last one better in particular.

It's nice to be able to grep for `ignore:` and `TODO` since they're generally a code smell, so killing those off is nice. Thanks!